.gitignore

```diff
@@ -0,0 +1,4 @@
+README.md
```

Comment:
`README.md`, `.stickler.yml`, `stylelint.config.js`  files should not be in `.gitignore`